src/fontra/views/editor/visualization-layer-definitions.js

```diff
@@ -1083,6 +1210,19 @@ function* iterPointsByIndex(path, pointIndices) {
   }
 }
 
+function* iterAnchorsPointsByIndex(anchors, anchorIndices) {
+  if (!anchorIndices || !anchors.length) {
+    return;
+  }
+  for (const index of anchorIndices) {
```

Comment:
Since anchors are compatible with points (have x and y properties), this can be simplified:
```js
  for (const index of anchorIndices) {
    yield anchors[index];
  }
```

Reply:
Thanks. Done.